python-template/{{cookiecutter.placeholder_repo_name}}/docs/conf.py

```diff
@@ -97,6 +97,16 @@ def get_version(*file_paths):
 project_title = '{{ cookiecutter.project_name }}'
 documentation_title = "{project_title}".format(project_title=project_title)
 
+
+# Set display_github to False if you don't want "edit on Github" button
```

Comment:
1. Presumably this is being added to the cookiecutter because it was already added and tested in a specific repo? If so, can you update the PR description with some links.  If not, maybe we should do that?
2. I would like us to switch to a default sphinx theme, because our custom theme has a lot of maintenance issues. If we do that, I think this is built in. But, maybe this already uses built in capabilities? Consider adding a link to the docs that detail use of this config.

Reply:
Yep, you are right. If we moved to default sphinx theme, it would automatically be added.

Reply:
Thanks. Looks fine, except that the tests aren't yet green. :)